### ISSUE TITLE
Website: Favicon not working

### ISSUE THREAD

**Issue by Adeesh-devanand** [gopaddle-io/configurator#47]
Can anyone who has previously used jekyll, mmistakes theme please help fix the [website](https://gopaddle-io.github.io/configurator/) favicon.

Here's the link to the [config.yml](https://github.com/gopaddle-io/configurator/blob/main/docs/_config.yml) file

Thank you for your help!!

**Comment by gopaddle-io** [comment]
Please send pull requests to SODACODE-2022

**Comment by aimanfatima** [comment]
Can I fix this? I believe only favicon.ico is missing in the assets/images folder. 
`<link rel="shortcut icon" type="image/x-icon" href="{{ site.baseurl }}/assets/images/favicon.ico">` https://github.com/gopaddle-io/configurator/blob/329f3c0ee731422be206a2a936de875226f6e67a/docs/_includes/head-custom.html
I've generated a favicon using https://realfavicongenerator.ne which I can place in the above folder, and it should work I guess?
The favicon generated looks like this - 
<img width="289" alt="image" src="https://user-images.githubusercontent.com/30265320/158052084-8ff41501-44af-42eb-8c8e-868839af5916.png">

**Comment by gopaddle-io** [comment]
Yes, you are right. We need a favicon as per the sample you had shared.

**Comment by Adeesh-devanand** [comment]
Hey @aimanfatima, sure go ahead and send a pull request. I will test it on my environment before merging the code.
